<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="cyclone1" name="org.molgenis.data.support.DynamicEntityTest" tests="5" failures="0" timestamp="6 Jun 2018 08:44:55 GMT" time="0.004" errors="0">
  <testcase name="setException" time="0.001" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setException" time="0.001" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setNoException" time="0.000" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setNoException" time="0.001" classname="org.molgenis.data.support.DynamicEntityTest"/>
  <testcase name="setException" time="0.001" classname="org.molgenis.data.support.DynamicEntityTest"/>
</testsuite> <!-- org.molgenis.data.support.DynamicEntityTest -->
